exec/cnex/array.c

```diff
@@ -100,11 +107,18 @@ Array *array_copyArray(Array *a)
     for (size_t i = 0; i < r->size; i++) {
         cell_copyCell(&r->data[i], &a->data[i]);
     }
+    // This is a new array, so we're going to force the ref count to 1.
```

Comment:
This refcount assignment seems to be redundant, because you just called `array_createArrayFromSize`, which initializes the refcount to 1.

Reply:
Removed unnecessary assignment and comment.